code/modules/mob/living/brain/posibrain.dm

```diff
@@ -1,5 +1,69 @@
 GLOBAL_VAR(posibrain_notify_cooldown)
 
+GLOBAL_LIST_EMPTY(on_station_posis)
+
+/datum/job/posibrain
+	title = JOB_NAME_POSIBRAIN
+	description = "Follow your AI's interpretation of your laws above all else, or your own interpretation if not connected to an AI. Choose one of many modules with different tools, ask robotics for maintenance and upgrades."
+	department_for_prefs = DEPT_BITFLAG_SILICON
+	department_head_for_prefs = JOB_NAME_AI
+	auto_deadmin_role_flags = DEADMIN_POSITION_SILICON
+	faction = "Station"
+	total_positions = 0
+	spawn_positions = 0
+	supervisors = "your laws" //No AI yet as you are just a cube
+	selection_color = "#ddffdd"
+	minimal_player_age = 21
+	exp_requirements = 120
+	exp_type = EXP_TYPE_CREW
+	random_spawns_possible = FALSE
+
+	display_order = JOB_DISPLAY_ORDER_CYBORG
+	departments = DEPT_BITFLAG_SILICON
+
+	show_in_prefs = FALSE //No reason to show in preferences
+
+/datum/job/posibrain/equip(mob/living/carbon/human/H, visualsOnly = FALSE, announce = TRUE, latejoin = FALSE, datum/outfit/outfit_override = null, client/preference_source = null)
```

Comment:
in job.DM line 549, you need to handle the case where the player isn't equipped. If you return null then this will add the job's antag rep without actually ever spawning the player.